### DISCUSSION TITLE
Using the "Docker Method Linux" setup but I don't want to use LetsEncrypt

### DISCUSSION THREAD

**Discussion by alanwilliams-YSD** [requarks/wiki#3880] [Help / Questions] [answered]
How can I access the config files to edit the setup for our custom SSL cert.  I have followed theses instructions:

https://docs.requarks.io/install/ubuntu

I stopped here:
Automatic HTTPS with Let's Encrypt (optional)

I don't see any paths in the regular config instructions, unless I'm missing something...
https://docs.requarks.io/install/config

What would be the best workaround for this?  We have our own wildcard cert that we want to use!

Any suggestions would be helpful.

OS: Ubuntu 20.04 and Docker Setup provided here: https://docs.requarks.io/install/ubuntu

**Comment by Phil-hostwiki** [comment, marked as answer]
Use Nginx reverse proxy and setup your custom SSL cert.
